src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_simple.hpp

```diff
@@ -125,7 +160,41 @@ class EltwiseLogLoopFunction : public SnippetsFunctionBase {
     std::shared_ptr<ov::Model> initOriginal() const override;
     std::shared_ptr<ov::Model> initReference() const override;
 };
-
+/// 2 results.
+/// So we have 2 subgraphs - Snippets don't support subgraphs with many results
+/// Also Output tensors have names to check correct copying output names
+//    in1    in2
+//    Sinh   Sinh
+//        Add
+//  HSwish   Result
+//  Relu
+//  Result
+class EltwiseTwoResultsFunction : public SnippetsFunctionBase {
+public:
+    explicit EltwiseTwoResultsFunction(const std::vector<Shape>& inputShapes) : SnippetsFunctionBase(inputShapes) {
+            NGRAPH_CHECK(input_shapes.size() == 2, "Got invalid number of input shapes");
+    }
+protected:
+    std::shared_ptr<ov::Model> initOriginal() const override;
+    std::shared_ptr<ov::Model> initReference() const override;
+};
+/// Two different Input and Outputs.
+/// This function is to check correct Broadcasting
+//        in1       in2
+//        Sin       Sin
+//       HSwish      /
+//  Result      Add
+//              Relu
+//              Sin
+//             Result
+class ManyInputsAndOutputsFunction : public SnippetsFunctionBase {
```

Comment:
Minor: Usually it's smth like 1,2,3,many , but here we have 1, many )))
Rename to`TwoInputsAndOutputsFunction` maybe?